CMakeLists.txt

```diff
@@ -55,7 +55,8 @@ add_definitions(-fPIC)
 
 # Baseline compiler flags, any change here will affect all build types
 #set (CMAKE_CXX_FLAGS "-Wall -Wno-unknown-pragmas -D_LARGEFILE64_SOURCE -D_GNU_SOURCE -Werror")
-set (CMAKE_CXX_FLAGS "-Wall -Wno-unknown-pragmas -D_LARGEFILE64_SOURCE -D_GNU_SOURCE -Werror -fno-var-tracking-assignments")
+set (CMAKE_CXX_FLAGS "-Wall -Wno-unknown-pragmas -D_LARGEFILE64_SOURCE -D_GNU_SOURCE -fno-var-tracking-assignments -I${CMAKE_INCLUDE_PATH}")
+set (CMAKE_EXE_LINKER_FLAGS "-L${CMAKE_LIBRARY_PATH}")
```

Comment:
Well, if you are installing on your system with all the default paths (installing on /usr), it will work. But if you want to install Orion and its dependencies somewhere else, it gives linker errors